### ISSUE TITLE
Native crashes on the CrystalSky (CS785U)

### ISSUE THREAD

**Issue by ericwarden** [dji-sdk/Mobile-SDK-Android#452]
Hey all,

We're seeing native crashes when initializing the DJI SDK on a CrystalSky (model CS785U). This appears to only happen on the CrystalSky as we also have a smart controller and it doesn't experience this issue. Also, it only appears to happen some of the time and it takes a few attempts to get it to reproduce. The crash appears during the `DJISDKManager.getInstance().registerApp` call.

This is all that the logcat shows:
```
W/linker: libsfjni.so: unused DT entry: type 0x6ffffffe arg 0x29d0
    libsfjni.so: unused DT entry: type 0x6fffffff arg 0x1
    
    --------- beginning of crash
A/libc: Fatal signal 4 (SIGILL), code 4, fault addr 0x9fbdb904 in tid 3551 (om.app.example)
A/libc: Fatal signal 11 (SIGSEGV), code 2, fault addr 0x3f795d5a in tid 3551 (om.app.example)
Process 3551 terminated.
```

and..
```
I/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
I/DEBUG: Build fingerprint: 'dji/zs600b/zs600b:5.1.1/v0.4.7.8-7da16f02/dji09291114:user/release-keys'
I/DEBUG: Revision: '0'
I/DEBUG: ABI: 'arm'
I/DEBUG: pid: 3551, tid: 3551, name: om.app.example  >>> com.app.example <<<
I/DEBUG: signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x3f795d5a
I/DEBUG:     r0 3f795d5a  r1 9fbe109c  r2 9fbe1000  r3 9fbe109c
I/DEBUG:     r4 9fbca48d  r5 b6f95477  r6 b45727e4  r7 b6f9553e
I/DEBUG:     r8 00000005  r9 b6f9a714  sl 00000001  fp 9fbe0760
I/DEBUG:     ip b6e6864c  sp bee31588  lr b6f8c5b7  pc 9fbdb930  cpsr 00070010
I/DEBUG: backtrace:
I/DEBUG:     #00 pc 00004930  <unknown>
I/DEBUG:     #01 pc 000015b5  /system/bin/linker (__dl__ZN6soinfo12CallFunctionEPKcPFvvE+44)
I/DEBUG:     #02 pc 00001689  /system/bin/linker (__dl__ZN6soinfo9CallArrayEPKcPPFvvEjb+140)
I/DEBUG:     #03 pc 0000185f  /system/bin/linker (__dl__ZN6soinfo16CallConstructorsEv+142)
I/DEBUG:     #04 pc 00003259  /system/bin/linker (__dl__Z9do_dlopenPKciPK17android_dlextinfo+192)
I/DEBUG:     #05 pc 00000f1d  /system/bin/linker (__dl__ZL10dlopen_extPKciPK17android_dlextinfo+24)
I/DEBUG:     #06 pc 001e0d89  /system/lib/libart.so (art::JavaVMExt::LoadNativeLibrary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, art::Handle<art::mirror::ClassLoader>, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >*)+544)
I/DEBUG:     #07 pc 00207b1b  /system/lib/libart.so (art::Runtime_nativeLoad(_JNIEnv*, _jclass*, _jstring*, _jobject*, _jstring*)+514)
I/DEBUG:     #08 pc 0007a745  /system/framework/arm/boot.oat
I/DEBUG: Tombstone written to: /data/tombstones/tombstone_08
```

I don't have proguard enabled for this build and I'm using DJI SDK version 4.10 with the following package options:
```
    packagingOptions {
        doNotStrip "*/*/libdjivideo.so"
        doNotStrip "*/*/libSDKRelativeJNI.so"
        doNotStrip "*/*/libFlyForbid.so"
        doNotStrip "*/*/libduml_vision_bokeh.so"
        doNotStrip "*/*/libyuv2.so"
        doNotStrip "*/*/libGroudStation.so"
        doNotStrip "*/*/libFRCorkscrew.so"
        doNotStrip "*/*/libUpgradeVerify.so"
        doNotStrip "*/*/libFR.so"

        doNotStrip "*/*/libDJIFlySafeCore.so"
        doNotStrip "*/*/libdjifs_jni.so"
        doNotStrip "*/*/libsfjni.so"
        exclude 'META-INF/rxjava.properties'
    }
```

Any help would be appreciated. 🙂

**Comment by dji-dev** [comment]
Public comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/29881'>Zendesk ticket #29881</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
This issue has been fixed in the latest v4.11.
Please update to the v4.11.

Thanks,

Luce Luo
DJI Dev Team

**Comment by ericwarden** [comment]
I haven't seen any crashes since updating to version 4.11. Thanks for the help!
